homeassistant/components/vicare/strings.json

```diff
@@ -97,13 +97,22 @@
         "name": "Comfort temperature"
       },
       "normal_heating_temperature": {
-        "name": "[%key:component::vicare::entity::number::normal_temperature::name%]"
+        "name": "Normal temperature (heating)"
```

Comment:
Open for better naming suggestions

Reply:
What does normal mean in this context? Default?

Default heating temperature?

Reply:
It's the program name: `normal`, `reduced/eco`, `comfort`, and there are some others (not yet integrated).